javascript/ql/src/Security/CWE-798/examples/HardcodedCredentialsHttpRequest.js

```diff
@@ -11,8 +11,8 @@ headers.append('Authorization', 'Basic' + base64.encode(username + ":" + passwor
 
 fetch(url, {method:'GET',
         headers: headers,
-        //credentials: 'user:passwd'
+        credentials: `${user}:${passwd}`
```

Comment:
Hmm. `user` and `passwd` are not declared variables in this example.  
Shouldn't you just delete the line with `credentials: ...`? 

(Same in the fixed example). 

---- 

And you still have the `//headers.append('Content-Type', 'text/json');` comment?